src/html-media-source.js

```diff
@@ -87,7 +89,9 @@ export default class HtmlMediaSource extends videojs.EventTarget {
       for (let i = 0; i < this.player_.audioTracks().length; i++) {
         let track = this.player_.audioTracks()[i];
 
-        if (track.enabled && track.kind !== 'main') {
+        if (track.enabled && track.kind === 'main') {
+          // The enabled track is a combined audio and video track so we'll enable the
+          // combined source buffer and disable the audio-only source buffer.
           combined = false;
```

Comment:
these need to be switched as well, maybe we should change audioDisabled_ to audioEnabled_ to make this whole thing easier?